ui/src/views/WorkerManager/WMViewWorkerPools/index.jsx

```diff
@@ -55,31 +77,58 @@ export default class WorkerManagerWorkerPoolsView extends Component {
           providerId: 'null-provider', // this is how we delete worker pools
         },
       },
-      refetchQueries: ['workerPools'],
+      refetchQueries: ['workerPoolsQuery'],
```

Comment:
I think this should be the name of the graphql query rather than the name of the variable. Looking inside the graphql, the name is `workerPools`.